pontoon/base/models.py

```diff
@@ -3442,3 +3454,14 @@ def serialize(self):
             "content": self.content,
             "id": self.id,
         }
+
+    def save(self, *args, **kwargs):
+        """
+        Validate Comments before saving.
+        """
+        if (self.translation and not self.locale and not self.entity) or (
+            not self.translation and self.locale and self.entity
+        ):
+            super(Comment, self).save(*args, **kwargs)
+        else:
+            raise ValidationError("Invalid comment arguments")
```

Comment:
Generally, it's better to reverse those conditions: first look for the failures, and let the normal behavior resolve outside your condition block. Since you're using a `raise` here, you don't need to `else` blocks, allowing to indent code a little less. I've also found it more natural to reason about. 

```py
if not (condition):
    raise ValidationError()

super(Comment, self).save()
```